### DISCUSSION TITLE
LangChain error: insufficient_quota

### DISCUSSION THREAD

**Discussion by gasperphoenix** [logancyang/obsidian-copilot#172] [General]
Hi, I created an API key, added it in the Copilot configuration (default model) and restarted the plugin. Unfortunately I get the error message "LangChain error: insufficient_quota" when writing a question.

How can I resolve it?

**Comment by logancyang** [comment]
Please refer to the FAQ section: https://github.com/logancyang/obsidian-copilot#-faq-please-read-before-submitting-an-issue
